presto-docs/src/main/sphinx/release/release-0.215.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.215
+=============
+
+General Changes
+---------------
+
+* Fix reporting of the processed input data for source stages in ``EXPLAIN ANALYZE``.
+* Fix bug where calculation of estimated block size can be quadratic. This fixes CPU waste for queries
+* Fix regression in 0.214 that could cause queries to produce incorrect results due to a bug in lazy hash table build for map types.
+* Improve spatial join performance by accelerating build-side geometries.
+* Extend stats and cost model to support ``ROW_NUMBER`` window function estimates.
+* Improve the automatic selection of join type and the reordering of join sides for cases where the join output size cannot be estimated.
+* Add dynamic scheduling support to grouped execution. When a stage is executed
+  with grouped execution and the stage has no remote sources, table partitions can be
+  scheduled to tasks in a dynamic way, which can help mitigating skew for queries using
+  grouped execution. This feature can be enabled with the
+  ``dynamic_schedule_for_grouped_execution`` session property or the
+  ``dynamic-schedule-for-grouped-execution`` config property.
+* Enable spatial joins for join conditions expressed with the :func:`ST_Within` function.
+* Add input validation checks to :func:`ST_LineString` to conform with the specification.
+* Add :func:`beta_cdf` and :func:`inverse_beta_cdf` functions.
+* Split the reporting of raw input data and processed input data for source operators.
+* Presto no longer reports raw input data for ``Values``, ``Local Exchange``, and ``Local Merge Sort`` operators.
+* Simplify ``EXPLAIN (TYPE IO)`` output when there are too many discrete components. This avoids large output
+  at the cost of reduced granularity.
+* Fail queries that use non-leaf resource groups to prevent them from getting stuck.
+* Add :func:`parse_presto_data_size` function.
+* Add support for ``UNION ALL`` to optimizer's cost model.
+* Add support for estimating the cost of filter nodes by using a default filter factor. The default value for the filter factor can be configured with
+  the ``default_filter_factor_enabled`` session property or the ``optimizer.default-filter-factor-enabled``.
+
+Web UI Changes
+--------------
+
+* Fix ``Capture Snapshot`` button for showing current thread stacks.
+* Fix drop-down to show stage skew bar chart.
+* Fix expand dropdown for stage skew component.
```

Comment:
Use consistent spelling across items: either drop-down or dropdown

Which drop-down? Is this a duplicate of the previous item?

Reply:
I think this is the same as "Fix expand dropdown for stage skew component." I agree we need info about which page, etc. this is affecting. /cc @raghavsethi